src/main/resources/application.yml

```diff
@@ -0,0 +1,31 @@
+skakater-nordmart-search:
+  kafka:
+    product.topic:
+      create.on-startup: true
+      name: productsTopic
+      number-of-partitions: 1
+      replication-factor: 1
+    bootstrap-servers: ${KAFKA_ADDRESS:localhost:9092}
+  cors.allowed.hosts: http://localhost:4200,http://0.0.0.0:4200
```

Comment:
Add an environment variable here as well, since CORS will block it when we'll deploy it on web(remotely against a certain hostname).

Reply:
Added